pkgs/data/themes/breath/default.nix

```
@@ -0,0 +1,47 @@
+{ mkDerivation
+, lib
+, fetchFromGitLab
+, gitUpdater
+, cmake
+, extra-cmake-modules
+, kdecoration
+, plasma-workspace
+, qtbase
+}:
+
+mkDerivation rec {
+  pname = "breath-theme";
+  version = "3130eb1addeebb4b93f1192c377383618ebecaa3";
+
+  src = fetchFromGitLab {
+    domain = "gitlab.manjaro.org";
+    owner = "themes";
+    group = "artwork";
+    repo = "breath";
+    rev = version;
+    sha256 = "1cc7zc31hhrzkcy6df1ckli2h4ij3lwclcm1wvcxsk24jdkr0hnb";
+  };
+
+  buildInputs = [
+    kdecoration
+    plasma-workspace
+    qtbase
+  ];
+
+  nativeBuildInputs = [
+    cmake
+    extra-cmake-modules
+  ];
+
+  cmakeFlags = [ "-DBUILD_PLASMA_THEMES=ON -DBUILD_SDDM_THEME=ON" ];
+
+  passthru.updateScript = gitUpdater { inherit pname version; };
+
+  meta = {
+    description = "Manjaro KDE default theme";
+    homepage = "https://gitlab.manjaro.org/artwork/themes/breath";
+    license = lib.licenses.cc-by-sa-40;
+    maintainers = [ lib.maintainers.huantian ];
+    platforms = lib.platforms.all;
+  };
```

```suggestion
  meta = with lib; {
    description = "Manjaro KDE default theme";
    homepage = "https://gitlab.manjaro.org/artwork/themes/breath";
    license = lib.licenses.cc-by-sa-40;
    maintainers = with maintainers; [ huantian ];
  };
```

I'm assuming by "all platforms" you meant only Linux and MacOS? If yes, this line is not required and setting `all` here will only lead to broken builds on other systems. Another question is: if KDE is required as an input, will this work on MacOS?

---

Hmm I didn't think about that, haven't tested on macOS. I don't think this would work macOS, in that case should I use `platforms = platforms.x86_64-linux;` instead?

---

My guess `platforms = platforms.linux` would be enough.